cmd/cleanup-controller/handlers.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"time"
+
+	"github.com/go-logr/logr"
+	"github.com/kyverno/kyverno/cmd/cleanup-controller/utils"
+	"github.com/kyverno/kyverno/cmd/cleanup-controller/validate"
+	"github.com/kyverno/kyverno/pkg/clients/dclient"
+	"github.com/kyverno/kyverno/pkg/openapi"
+	admissionutils "github.com/kyverno/kyverno/pkg/utils/admission"
+	admissionv1 "k8s.io/api/admission/v1"
+)
+
+type cleanupPolicyHandlers struct {
+	client         dclient.Interface
+	openApiManager openapi.Manager
```

Comment:
Why add openAPIManager? Do we need extra schema validation for cleanup policies?

Reply:
Aren't we going to need it to validate the resource scope ?